docs/osf-trivia-list/dasharo.md

```diff
@@ -154,3 +154,83 @@ Dasharo is not coreboot firmware distribution because we also may imagine
 Dasharo pure EDKII, oreboot or based on any other open-source firmware
 framework. Dasharo typically is based on coreboot and EDKII, plain coreboot is
 not enough in most cases for booting modern computer.
+
+## What value Dasharo provide in comparison to coreboot?
+
+* Dasharo is open-source firmware distribution based on
+  [coreboot](https://coreboot.org) and other open-source firmware frameworks
+  (e.g. [Tianocore EDKII](https://github.com/tianocore/edk2)), you can think
+  about Dasharo and coreboot relation in the same way as you think about
+  Debian/Ubuntu/RedHat and Linux. That means it can provide better
+  cost-effectiveness, security, transparency and customizability than
+  proprietary alternative.
+* There are seven characterestics of Dasharo:
+    - [zero-touch initial deployment](#dasharo-zero-touch-initial-deployment),
+    - [clean and simple code](#dasharo-clean-and-simple-code),
+    - [long-term maintenance](#dasharo-long-term-maintenance),
+    - [professional support](#dasharo-support-package),
+    - [transparent validation](#dasharo-transparent-validation),
+    - [privacy-respecting implementation](#dashro-privacy-respecting-implementation),
+    - [liberty for the owners](#dasharo-liberty-for-the-owners),
```

Comment:
This is mentioned and linked here, but not introduced in this MR.
OTOH, it somehow described in another MR: https://github.com/Dasharo/docs/pull/507/files#diff-c8be10a4380efbce4c82013c5d1e8d7d42423eda7107a96990f0c54284767b2cR301

Reply:
a/a https://github.com/Dasharo/docs/pull/504/commits/a51ca4d68bd9f339f71f23e3edc75f34c21f3018